src/styles/blocks/wrapper.scss

```diff
@@ -0,0 +1,5 @@
+.wrapper {
+  display: flex;
+  justify-content: center;
+  padding: 100px;

```

Comment:
its hardcode vertical positioning ) use height: 100vh; and align-items: center;

Reply:
Make sense, thanks ;)